package.json

```diff
@@ -24,6 +24,7 @@
     "build:bazel": "node ./bin/devkit-admin build-bazel",
     "build-tsc": "tsc -p tsconfig.json",
     "lint": "eslint --cache --max-warnings=0 \"**/*.ts\"",
+    "ng-dev": "TS_NODE_PROJECT=$PWD/.ng-dev/tsconfig.json node_modules/@angular/dev-infra-private/ng-dev/bundles/cli.mjs",
```

Comment:
Note: We cannot dynamically set the tsconfig as ts-node/esm needs to be registered before runtime. The explicit tsconfig is needed for `ts-node` to load the `.ng-dev` config as ESM

Reply:
You sure about that? I believe that if you change the shebang of `node_modules/@angular/dev-infra-private/ng-dev/bundles/cli.mjs` to `#!/usr/bin/env ts-node-esm` it should work.

Reply:
@alan-agius4 It's already doing that, notice how we don't run `ts-node` here. The deciding part is that we would need to tell `ts-node` about the `tsconfig` before runtime as well (hence the environment variable)

Reply:
Oh I didn't notice that.